src/styles/pages/admin/_rewrites.scss

```diff
@@ -515,6 +515,17 @@ button[class*="c-button"].c-checkbox-dropdown-modal__trigger {
 // dropdowns
 
 .c-filter-table__toolbar {
+	.c-dropdown__trigger {
+		.c-checkbox-dropdown-modal__trigger {
+			padding: 0.54rem $spacer-md;
+		}
+
+		[class^="TagList-module_c-tag-list"] {
```

Comment:
Will this selector work after a PRD build's class obfuscation?

Reply:
yes, the class always starts the same way:
![image](https://user-images.githubusercontent.com/1710840/174613539-31bdeff1-df2d-4163-8719-367bc54504c8.png)